code/modules/admin/verbs/adminevents.dm

```diff
@@ -34,33 +34,42 @@
 	admin_headset_message(M)
 
 /client/proc/admin_headset_message(mob/M in GLOB.mob_list, sender = null)
```

Comment:
by the way can you change this single letter var out to something like `mob/target`. i don't think anyone else will touch this code for a while and it would be a great help for readability